src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -129,7 +130,8 @@ const ValueElement = (props) => {
     overridableId,
   } = props;
 
-  const label = valueLabel ? valueLabel : `${bucket.key} (${bucket.doc_count})`;
+  const label = valueLabel ? `${valueLabel} (${bucket.doc_count}) `: 
```

Comment:
I think that the `valueLabel` should be able to override the whole value i.e not force the `doc_count` to appear! I would say that we can go with 2 solutions:
1) Compute the `${valueLabel} (${bucket.doc_count}) ` beforehand when we build the value map or
2) Create another prop e.g 
```javascript
const templateLabel = (valueLabel, bucket) => valueLabel ? `${valueLabel} (${bucket.doc_count})` : `${bucket.key} (${bucket.doc_count})`
```
so then the above line would be something like
```javascript
const label = props.templateLabel(valueLabel, bucket);
```

In this way, you have property that can construct your final template based on the bucket! Your `valueLabels` map can remain still plain values! @ntarocco WDYT? 

**Context**: We need this to override the text of nested aggregations in RDM. See https://github.com/inveniosoftware/invenio-rdm-records/issues/202